### PR TITLE
feat(nextjs): error reporting and tracing

### PR DESCRIPTION
**Pull request by sambacha**

### CODE REVIEW

**Comment by sambacha** [comment]
@matthewlilley

File: next.config.js [hunk 1]
```diff
@@ -76,6 +116,14 @@ const nextConfig = {
       },
     ]
   },
+  async headeres() {
```
Comment (matthewlilley) [inline next.config.js:119]: Looks like a typo

**Review by matthewlilley — CHANGES_REQUESTED** [review]
Looks like the multicall files included are uneeded now? Besides that this looks good.

File: next.config.js (hunk 1 above)
Comment (sambacha) [reply next.config.js]: https://github.com/sushiswap/sushiswap-interface/pull/821

**Comment by sambacha** [comment]
closing in favour of https://github.com/sushiswap/sushiswap-interface/pull/821

**Pull request closed without merging**